inc/wp-graphql.php

```diff
@@ -257,3 +257,44 @@ function register_headless_settings( $type_registry ) {
 	);
 }
 add_action( 'graphql_register_types', __NAMESPACE__ . '\register_headless_settings' );
+
+/**
+ * Register Gravatar URL with GraphQL.
+ *
+ * @author WebDevStudios
+ * @since 2.0.0
```

Comment:
Should we bump this to 2.1.0 and update the plugin version to match?

Reply:
Done and yes, w're going to need to tag and release `2.1.0`